client/app/bundles/course/assessment/pages/AssessmentStatistics/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { getAssessmentStatistics } from './selectors';
+import StudentMarksPerQuestionTable from './StudentMarksPerQuestionTable';
+
+const translations = defineMessages({
+  statistics: {
+    id: 'course.assessment.statistics.statistics',
+    defaultMessage: 'Statistics',
+  },
+  header: {
+    id: 'course.assessment.statistics.header',
+    defaultMessage: 'Statistics for {title}',
+  },
+  fetchFailure: {
+    id: 'course.assessment.statistics.fail',
+    defaultMessage: 'Failed to fetch statistics.',
+  },
+  fetchAncestorsFailure: {
+    id: 'course.assessment.statistics.ancestorFail',
+    defaultMessage: 'Failed to fetch past iterations of this assessment.',
+  },
+  fetchAncestorStatisticsFailure: {
+    id: 'course.assessment.statistics.ancestorStatisticsFail',
+    defaultMessage: "Failed to fetch ancestor's statistics.",
+  },
+  duplicationHistory: {
+    id: 'course.assessment.statistics.duplicationHistory',
+    defaultMessage: 'Duplication History',
+  },
+  marksPerQuestion: {
+    id: 'course.assessment.statistics.marksPerQuestion',
+    defaultMessage: 'Marks Per Question',
+  },
+  gradeDistribution: {
+    id: 'course.assessment.statistics.gradeDistribution',
+    defaultMessage: 'Grade Distribution',
+  },
+  submissionTimeAndGrade: {
+    id: 'course.assessment.statistics.submissionTimeAndGrade',
+    defaultMessage: 'Submission Time and Grade',
+  },
+  includePhantom: {
+    id: 'course.assessment.statistics.includePhantom',
+    defaultMessage: 'Include Phantom Student',
+  },
+});
+
+const AssessmentStatisticsPage: FC = () => {
+  const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState('marksPerQuestion');
+  const [includePhantom, setIncludePhantom] = useState(false);
+
+  const { assessmentId } = useParams();
+  const parsedAssessmentId = parseInt(assessmentId!, 10);
+  const dispatch = useAppDispatch();
+
+  const statistics = useAppSelector(getAssessmentStatistics);
+
+  useEffect(() => {
+    dispatch(actions.reset());
+    dispatch(fetchAssessmentStatistics(parsedAssessmentId)).catch(() =>
+      toast.error(t(translations.fetchFailure)),
+    );
+  }, [dispatch]);
+
+  const tabComponentMapping = {
+    marksPerQuestion: (
+      <StudentMarksPerQuestionTable includePhantom={includePhantom} />
+    ),
+    gradeDistribution: <MainGradesChart includePhantom={includePhantom} />,
+    submissionTimeAndGrade: (
+      <MainSubmissionTimeAndGradeStatistics includePhantom={includePhantom} />
+    ),
+    duplicationHistory: <DuplicationHistoryStatistics />,
+  };
+
+  if (statistics.isLoading) {
+    return <LoadingIndicator />;
+  }
+
+  return (
+    <Page
+      backTo={statistics.assessment?.url}
+      className="space-y-5"
+      title={t(translations.header, {
+        title: statistics.assessment?.title ?? '',
+      })}
+    >
+      <>
+        <Box className="max-w-full border-b border-divider">
+          <SubmissionStatusMainAssessment includePhantom={includePhantom} />
```

Comment:
Can we rename this to SubmissionBarChart?

Reply:
I don't think this is viable, because:
- even though it's rendering BarChart, we want to emphasise that the chart is actually showing the status (workflowState); it just happens that the component we're using is BarChart
- we have another component called SubmissionStatusChart which might be of your interest regarding this comment. However, this component is used for both MainAssessment (which has the dependency towards the state includePhantom) and AncestorAssessment (independent of the mentioned state)